recipes-domx/meta-xt-images-domx/recipes-devtools/python/python-clang_5.0.post2.bb

```diff
@@ -0,0 +1,2 @@
+inherit setuptools
+require python-clang.inc
```

Comment:
What it the reason having separate file all the configs are defined in the `.inc` file?